<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>834</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Noto Color Emoji</family>
    <pointsize>72</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
background-color: transparent;
border-image: url(:etc/bimgs_qt/menu_backgroung.jpg);
background: none;
border: none;
background-repeat: none;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_record">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>641</width>
      <height>481</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_record{
background-color: transparent;
border-image: url(:etc/bimgs_qt/WARMING.jpg);
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="square">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>-50</y>
      <width>851</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#square{
background-color: transparent;
border-image: url(:etc/bimgs_qt/SQUARE_SHAPE.png);
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Number of faces:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="arrowleft_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>41</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#arrowleft_button{
background-color: transparent;
border-image: url(:etc/bimgs_qt/ARROW_LEFT.png);
background: none;
border: none;
background-repeat: none;
}
#arrowleft_button:pressed{
background-color: transparent;
border-image: url(:etc/bimgs_qt/ARROW_LEFT_on.png);
background: none;
border: none;
background-repeat: none;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="arrowright_button">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>270</y>
      <width>41</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#arrowright_button{
background-color: transparent;
border-image: url(:etc/bimgs_qt/ARROW_RIGHT.png);
background: none;
border: none;
background-repeat: none;
}
#arrowright_button:pressed{
background-color: transparent;
border-image: url(:etc/bimgs_qt/ARROW_RIGHT_on.png);
background: none;
border: none;
background-repeat: none;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>460</y>
      <width>160</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <widget class="QPushButton" name="exit_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>131</width>
          <height>141</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#exit_button{
background-color: transparent;
border-image: url(:etc/bimgs_qt/EXIT.png);
background: none;
border: none;
background-repeat: none;
}
#exit_button:pressed{
background-color: transparent;
border-image: url(:etc/bimgs_qt/EXIT_on.png);
background: none;
border: none;
background-repeat: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>90</y>
      <width>160</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <widget class="QPushButton" name="play_button">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>111</width>
          <height>101</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#play_button{
background-color: transparent;
border-image: url(:etc/bimgs_qt/PLAY.png);
background: none;
border: none;
background-repeat: none;
}
#play_button:pressed{
background-color: transparent;
border-image: url(:etc/bimgs_qt/PLAY_on.png);
background: none;
border: none;
background-repeat: none;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>141</width>
          <height>121</height>
         </rect>
        </property>
        <widget class="QPushButton" name="settings_button">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>101</width>
           <height>101</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">#settings_button{
background-color: transparent;
border-image: url(:etc/bimgs_qt/SETTINGS.png);
background: none;
border: none;
background-repeat: none;
}
#settings_button:pressed{
background-color: transparent;
border-image: url(:etc/bimgs_qt/SETTINGS_on.png);
background: none;
border: none;
background-repeat: none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLCDNumber" name="number_of_faces_lcd">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>161</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>72</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="IDLE_MODE_LABEL">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>670</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#IDLE_MODE_LABEL{
background-color: transparent;
border-image: url(:etc/bimgs_qt/LIGHT_MODE.png);
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="MANUAL_MODE_LABEL">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>670</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#MANUAL_MODE_LABEL{
background-color: transparent;
border-image: url(:etc/bimgs_qt/LIGHT_MODE.png);
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="DETECTION_MODE_LABEL">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>670</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#DETECTION_MODE_LABEL{
background-color: transparent;
border-image: url(:etc/bimgs_qt/LIGHT_MODE.png);
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="RECOGNITION_MODE_LABEL">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>670</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#RECOGNITION_MODE_LABEL{
background-color: transparent;
border-image: url(:etc/bimgs_qt/LIGHT_MODE.png);
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LIGHT_IDLE_MODE">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>660</y>
      <width>67</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#LIGHT_IDLE_MODE{
background-color: transparent;
border-image: url(:etc/bimgs_qt/LIGHT_OFF.png);
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LIGHT_MANUAL_MODE">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>660</y>
      <width>67</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#LIGHT_MANUAL_MODE{
background-color: transparent;
border-image: url(:etc/bimgs_qt/LIGHT_OFF.png);
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LIGHT_DETECTION_MODE">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>660</y>
      <width>67</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#LIGHT_DETECTION_MODE{
background-color: transparent;
border-image: url(:etc/bimgs_qt/LIGHT_OFF.png);
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LIGHT_RECOGNITION_MODE">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>660</y>
      <width>67</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#LIGHT_RECOGNITION_MODE{
background-color: transparent;
border-image: url(:etc/bimgs_qt/LIGHT_OFF.png);
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>640</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>IDLE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>640</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>MANUAL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>640</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DETECTION</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>640</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RECOGNITION</string>
    </property>
   </widget>
   <widget class="QPushButton" name="arrowup_button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#arrowup_button{
background-color: transparent;
border-image: url(:etc/bimgs_qt/ARROW_UP.png);
background: none;
border: none;
background-repeat: none;
}
#arrowup_button:pressed{
background-color: transparent;
border-image: url(:etc/bimgs_qt/ARROW_UP_on.png);
background: none;
border: none;
background-repeat: none;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="arrowdown_button">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>560</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#arrowdown_button{
background-color: transparent;
border-image: url(:etc/bimgs_qt/ARROW_DOWN.png);
background: none;
border: none;
background-repeat: none;
}
#arrowdown_button:pressed{
background-color: transparent;
border-image: url(:etc/bimgs_qt/ARROW_DOWN_on.png);
background: none;
border: none;
background-repeat: none;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>LIGHT_RECOGNITION_MODE</zorder>
   <zorder>LIGHT_DETECTION_MODE</zorder>
   <zorder>LIGHT_MANUAL_MODE</zorder>
   <zorder>LIGHT_IDLE_MODE</zorder>
   <zorder>square</zorder>
   <zorder>label_record</zorder>
   <zorder>label</zorder>
   <zorder>arrowleft_button</zorder>
   <zorder>arrowright_button</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>number_of_faces_lcd</zorder>
   <zorder>RECOGNITION_MODE_LABEL</zorder>
   <zorder>DETECTION_MODE_LABEL</zorder>
   <zorder>MANUAL_MODE_LABEL</zorder>
   <zorder>IDLE_MODE_LABEL</zorder>
   <zorder>label_2</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>arrowup_button</zorder>
   <zorder>arrowdown_button</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exit_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1040</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>1238</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
